scrap.py

```diff
@@ -37,7 +37,10 @@ def check_connection(self) -> bool:
             raise ConnectionError(f"{self.url}: is Not reachable \nErr: {e}")
 
     # it's static function now later with database I will make it related to this class
-    def resume_scarping(self, f_posts: dict, last_post: dict) -> bool:
+
+    @staticmethod
```

Comment:
Why did you change this to a static method?

Reply:
it's already static I just confirmed by adding the decorator